projects/vc1x/vc1x-version.props

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup Label="UserMacros">
-    <VersionString>2_60</VersionString>
+    <VersionString>2_06</VersionString>

```

Comment:
Is this right?

Reply:
Yeah, not very intuitive, which is probably why I got it wrong the first time!